django_ledger/contrib/django_ledger_graphql/accounts/schema.py

```diff
@@ -0,0 +1,22 @@
+
+import graphene
+from graphene_django import DjangoObjectType
+from django_ledger.models import AccountModel
+
+
+class AccountList(DjangoObjectType):
```

Comment:
I believe this should be named AccountModelGQLBase or similar since this is a wrapper around the Django Model. Naming should be consistent throughout all models.

Reply:
Actually, according to documentation these should be called Nodes. i.e AccountModelNode

Reply:
i dont think it's necessary, if am not wrong that convection is used when we want  filtering fuctionality something like this 
`class AccountModelNode(DjangoObjectType):
    class Meta:
        model = AccountModel
        filter_fields = ['name', 'active']
        interfaces = (relay.Node, )`
https://docs.graphene-python.org/projects/django/en/latest/filtering/